api/server.js

```diff
@@ -2,9 +2,12 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const cookieParser = require('cookie-parser');
 const path = require('path');
+const mongoose = require('./db/mongoose');
 
 const app = express();
 
+const secret = 'mysecretsshhh';
```

Comment:
Variable is unused

Reply:
I also didn't understand what was the point of this variable... Just copy from the boilerplate...

Reply:
I will delete it!